vizro-core/docs/pages/user-guides/data.md

```diff
@@ -1,199 +1,39 @@
-        )
-
-        dashboard = vm.Dashboard(pages=[page])
-
-        Vizro().build(dashboard).run()
-        ```
-    === "app.py (use functools.partial)"
-        ```py linenums="1"
-        from functools import partial
-
-        from vizro import Vizro
-        import vizro.plotly.express as px
-        import vizro.models as vm
-        from vizro.managers import data_manager
-
-
-        def retrieve_one_species(species):
-            df = px.data.iris()
-            subset = df[df["species"] == species].copy()
-            return subset
-
-
-        data_manager["species_setosa"] = partial(retrieve_one_species, "setosa")
-
-        page = vm.Page(
-            title="My first page",
-            components=[
-                vm.Graph(id="scatter_chart", figure=px.scatter("species_setosa", x="sepal_length", y="petal_width", color="species")),
-            ],
-            controls=[vm.Filter(column="species")],
-        )
-
-        dashboard = vm.Dashboard(pages=[page])
-
-        Vizro().build(dashboard).run()
-        ```
-    === "Result"
-        [![DataConnector]][DataConnector]
-
-    [DataConnector]: ../../assets/user_guides/data/data_selected_from_source.png
-
-
-### Kedro integration
-
-If the data you are visualizing is a [`kedro_datasets.pandas`](https://docs.kedro.
-org/en/stable/kedro_datasets.html) type from a Kedro data catalog, you can use
-Vizro's [Kedro integration](integration.md#kedro) to connect your charts to [Kedro's Data Catalog](https://docs.kedro.org/en/stable/data/index.html).
-
-If it is not a `kedro_datasets.pandas` type, you need to build a
-data connector to load the data from the data catalog and convert it to a Pandas
-DataFrame, before you can register it with Vizro Data Manager.
+Vizro supports two different types of data:
+
+* [Static data](static-data.md): pandas DataFrame. This is the simplest method and best to use if you do not need the more advanced functionality of dynamic data.
+* [Dynamic data](dynamic-data.md): function that returns a pandas DataFrame. This is a bit more complex to understand but has more advanced functionality such as the ability to refresh data while the dashboard is running.
+
+The following flowchart shows what you need to consider when choosing how to set up your data.
+``` mermaid
+graph TD
+  refresh["`Do you need your data to refresh while the dashboard is running?`"]
+  specification["`Do you need to specify your dashboard through a configuration language like YAML?`"]
+  dynamic([Use dynamic data referenced by name])
+  static-direct([Use static data supplied directly])
+  static-name([Use static data referenced by name])
+
+  refresh -- No --> specification
+  refresh -- Yes --> dynamic
+  specification -- No --> static-direct
+  specification -- Yes --> static-name
+  
+  click static-direct href "../static-data#supply-directly"
+  click static-name href "../static-data#reference-by-name"
+  click dynamic href "../dynamic-data"
```

Comment:
These are clickable but not obvious to the user unless they mouseover. Unfortunately just tinkering with the CSS doesn't help here, we need to wait for progress to be made on https://github.com/squidfunk/mkdocs-material/issues/5619 which I've subscribed to.

Reply:
Oh nice, thank you! I don't know how I missed that in the Mermaid docs because I read through them like a million times 😅 Let me play around with it tomorrow

Reply:
Done in in a slightly neater way in [229f4c1](https://github.com/mckinsey/vizro/pull/398/commits/229f4c1fe5f98ad20714843a80b3c12b99d077ce).